From 288ef6585150f2a58d4c1723efae1a5a7a41772a Mon Sep 17 00:00:00 2001
From: Zhenyu Wang <zhenyu.z.wang@intel.com>
Date: Tue, 20 May 2008 08:54:01 +0800
Subject: [PATCH] Revert "Add FIFO watermark regs to register dumper"

This reverts commit 0c00a638ef57aa9d6a3047176b0bfad733f781f0.

Those FIFO watermark regs are 945-ish, and cause problem
on G35.
(cherry picked from commit 740a73dad1d9a9b0ebf1cc7d6f2a056abad57aaf)

diff --git a/src/i810_reg.h b/src/i810_reg.h
index cc4620a..834b948 100644
--- a/src/i810_reg.h
+++ b/src/i810_reg.h
@@ -497,7 +497,6 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 #define FWATER_BLC       0x20d8
 #define FWATER_BLC2	 0x20dc
-#define FWATER_BLC_SELF	 0x20e0
 #define MM_BURST_LENGTH     0x00700000
 #define MM_FIFO_WATERMARK   0x0001F000
 #define LM_BURST_LENGTH     0x00000700
diff --git a/src/i830_debug.c b/src/i830_debug.c
index 0eba57f..15b02ce 100644
--- a/src/i830_debug.c
+++ b/src/i830_debug.c
@@ -546,10 +546,6 @@ static struct i830SnapshotRec {
     DEFINEREG(FBC_FENCE_OFF),
     DEFINEREG(FBC_MOD_NUM),
 
-    DEFINEREG(FWATER_BLC),
-    DEFINEREG(FWATER_BLC2),
-    DEFINEREG(FWATER_BLC_SELF),
-
     DEFINEREG2(FPA0, i830_debug_fp),
     DEFINEREG2(FPA1, i830_debug_fp),
     DEFINEREG2(DPLL_A, i830_debug_dpll),
-- 
1.5.5.1

